README.md

```diff
@@ -3,8 +3,12 @@ SPDX-FileCopyrightText: 2022 Contributors to the Power Grid Model IO project <dy
 
 SPDX-License-Identifier: MPL-2.0
 -->
+[![License: MIT](https://img.shields.io/badge/License-MPL2.0-informational.svg)](https://github.com/alliander-opensource/power-grid-model/blob/main/LICENSE)
```

Comment:
@Jonasvdbo I think Bram meant this by references.  Should point to power-grid-model-io